examples/examples/valuable.rs

```diff
@@ -0,0 +1,69 @@
+#![allow(dead_code)]
+//! This example shows how a value may be recorded using valuable and as an alternative recording
+//! it without valuable but just using the debug printout. Valuable is more flexible however, as
+//! a user can implement a custom `tracing_core::Visit` and implement
+//! `tracing_core::Visit::record_value` to handle the valuable and extract the fields, whereas with
+//! the debug printout you'd need to parse the string to extract the values.
+#[cfg(tracing_unstable)]
+use tracing::field::valuable;
+use tracing::{info, info_span};
+#[cfg(tracing_unstable)]
+use valuable::Valuable;
+
+#[cfg_attr(tracing_unstable, derive(Valuable))]
+#[derive(Clone, Debug)]
+struct User {
+    name: String,
+    age: u32,
+    address: Address,
+}
+
+#[cfg_attr(tracing_unstable, derive(Valuable))]
+#[derive(Clone, Debug)]
```

Comment:
i think we can have the example always depend on `valuable`, and always derive the trait --- it's specifically *tracing's* `valuable` support that's feature-flagged. that should make this look a little cleaner:
```suggestion
use valuable::Valuable;

#[derive(Valuable, Clone, Debug)]
struct User {
    name: String,
    age: u32,
    address: Address,
}

#[derive(Valuable, Clone, Debug)]
```